src/net/async/host.cpp

```diff
@@ -0,0 +1,181 @@
+/*
```

Comment:
If you don't want to add something, then just don't add it. Don't add commented out files.